package.json

```diff
@@ -14,6 +19,7 @@
   },
   "license": "AGPL-3.0",
   "dependencies": {
+    "@storybook/addon-docs": "^6.2.1",
```

Comment:
Shouldn't this be under `devDependencies`? :thinking:

Reply:
Actually, it's not that easy, because: 
 - `dependencies`: Packages required by your application in production.
 - `devDependencies`: Packages that are only needed for local development and testing.

StoryBook can be run in production, to show the components used, but is clearly used for development.
I don't have a strong opinion on this. I will do what you think is more logical with the OFN standard.

Reply:
I think we don't want to load all this extra code in production, if possible :+1: